src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars--4 {
+  display: flex;
+
+  .stars__star {
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-position: center;
+  }
+
+  & :nth-child(-n + 4) {
```

Comment:
Now you only have a selector to show 4 active stars. But you should write selectors for all possible variants (from 0 to 5). Use the SCSS loop for it. If you need help feel free to ask for it in the chat

![image](https://github.com/user-attachments/assets/e36de2c7-f5a2-4a63-8e06-8060bb302647)